coordinates/sites.json

```
@@ -748,34 +748,34 @@
         ]
     },
     "dkist": {
-	"aliases": [
-	    "DKIST",
-	    "ATST"
-	],
-	"name": "Daniel K. Inouye Solar Telescope",
-	"elevation": 3104,
-	"elevation_unit": "meter",
-	"latitude": 20.7047,
-	"latitude_unit": "degree",
-	"longitude": 203.8233,
-	"longitude_unit": "degree",
-	"timezone": "US/Aleutian",
-	"source": "DKIST website: http://dkist.nso.edu/faq"
+        "name": "Daniel K. Inouye Solar Telescope",
+        "elevation": 3104,
+        "elevation_unit": "meter",
+        "latitude": 20.7047,
+        "latitude_unit": "degree",
+        "longitude": 203.8233,
+        "longitude_unit": "degree",
+        "timezone": "US/Aleutian",
```

```suggestion
        "timezone": "US/Hawaii",
```
see #117 - should be either this or `HST`, but I think discussion there leans towards `US/Hawaii`.